packages/webdriverio/src/commands/element/moveTo.ts

```diff
@@ -1,4 +1,4 @@
-import { getElementRect, getScrollPosition, getBrowserObject } from '../../utils/index.js'
```

Comment:
We should get rid of `getScrollPosition` as it is nowhere used no more.

Reply:
Ok i'll create some test and get rid of getScrollPosition

Reply:
Could you look into it https://github.com/webdriverio/guinea-pig/pull/2. There is an extra page for moveTo checks
Is it correct project for resource http://guinea-pig.webdriver.io ?

Reply:
@udarrr I merged the PR and the guinea pig page was deployed

Reply:
@christian-bromann  I was able to add some test, please check it.